### ISSUE TITLE
NTR: Bone biopsy: granulomas (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9206]
New term request Bone biopsy: granulomas (elthat_101120200048):

**Comment by pnrobinson** [comment]
@McikeySegal

Adding new term: 
- ID: HP:6000154
- label: Bone marrow granuloma
- parents: [Abnormal bone marrow cell morphology (HP:0005561)](https://hpo.jax.org/app/browse/term/HP:0005561)
- pmid: PMID:32119473
- def: A granuloma (a focal aggregate of immune cells that forms in response to a persistent inflammatory stimulus) localiyed to the bone marrow. A granuloma characteristically demonstrates the compact organization of mature macrophages, which may or may not be associated with other inflammatory cell types.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9206
- synonyms:
